src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -764,6 +764,8 @@ protected void finalize () {
      * @param  data the specified ICC Profile data
      * @return an {@code ICC_Profile} object corresponding to the data in the
      *         specified {@code data} array
+     * @throws IllegalArgumentException If the byte array does not contain valid
```

Comment:
The spec does already say "Throws an {@code IllegalArgumentException} if the data does		     
    * not correspond to a valid ICC Profile"
So really you are just adding the @throws clause. 
You might want to consider removing the now redundant text.

Reply:
Agree. that text is removed.